docs/source/user_guide/data.rst

```diff
@@ -30,3 +30,90 @@ Usage
 .. code-block:: python
 
     model = ModelData.download('http://yoururl.test/model', architecture='mace', filename='model', cache_dir='/home/mlip/', force_download=False)
+
+-The architecture of the model file can be accessed using the `architecture` property:
+
+.. code-block:: python
+
+    model_arch = model.architecture
+
+
+
+-The filepath of the model file can be accessed using the `filepath` property:
+
+.. code-block:: python
+
+    file_path = model.filepath
+
+.. warning::
+
+    When sharing data, using the ``filepath`` could point to a location inaccessible on another computer.
+    So if you are using data from someone else, for both the modeldata and the configfile, consider using the ``get_content()`` method to create a new file with identical content.
+    Then, use the filepath of the newly created file for running calculation.
+    A more robust solution to this problem is going to be implemented.
+
+
+JanusConfigfile
```

Comment:
Wondering if this documentation should be here or in the generated docs and docstrings with just a simple link to the API docs here

Reply:
I wonder the same, and I can (should?) add more info in the docstrings for the apidoc.
However, as a beginner/lazy user with little programming experience, I do remember finding api documentations confusing and preferring when everything was also written down in some user guide in a format with some examples.

If you are referring more specifically to the explanations for  `store_all` and `skip` that should definitely go in the api docs, maybe in a shorter format,  I put it here because it seemed a bit long.

Reply:
Probably create an issue and get this in